db/migrate/20210405210045_make_campaign_distributor_id_non_null.rb

```diff
@@ -0,0 +1,5 @@
+class MakeCampaignDistributorIdNonNull < ActiveRecord::Migration[6.0]
```

Comment:
maybe also run `rubocop -a` on these new files

Reply:
Went a little overboard and ran rubocop on all the files sooooo oops. But resolved